linkerd2-proxy/Cargo.toml

```diff
@@ -8,12 +8,8 @@ publish = { workspace = true }
 description = "The main proxy executable"
 
 [features]
-default = ["meshtls-rustls-aws-lc"]
-meshtls-boring = ["linkerd-meshtls/boring"]
-meshtls-boring-fips = ["linkerd-meshtls/boring-fips"]
-meshtls-rustls-aws-lc = ["linkerd-meshtls/rustls-aws-lc"]
-meshtls-rustls-aws-lc-fips = ["linkerd-meshtls/rustls-aws-lc-fips"]
-meshtls-rustls-ring = ["linkerd-meshtls/rustls-ring"]
```

Comment:
it looks like this PR should be based on the branch that removes ring... i'd make the same recommendation that this leaves the existing aws-featuers as they are.